daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/DFDL_part2_attributes.xsd

```diff
@@ -729,7 +729,7 @@
     Text representation -->
 
   <xsd:attributeGroup name="TextBidiSchemeAGQualified">
-    <xsd:attribute form="qualified" name="textBidi" type="dfdl:YesNoEnum" />
+    <!-- <xsd:attribute form="qualified" name="textBidi" type="dfdl:YesNoEnum" /> -->
```

Comment:
Should these commented out attributes be removed?

Reply:
Will delete. These didn't conflict with the primary definition because it was misspelled as BiDi with capital D. These were not so misspelled. Once fixed, they conflict. TextBidi is in common mixin, so doesn't need to be here in these attribute groups.